{{cookiecutter.project_slug}}/docker/centos/Dockerfile

```diff
@@ -0,0 +1,123 @@
+FROM {{ cookiecutter.docker_base }} as base
+#
+#  USAGE:
+#     cd services/{{ cookiecutter.project_slug }}
+#     docker build -f Dockerfile -t {{ cookiecutter.project_slug }}:prod --target production ../../
+#     docker run {{ cookiecutter.project_slug }}:prod
+#
+
+LABEL maintainer={{ cookiecutter.git_username }}
+
+# simcore-user uid=8004(${SC_USER_NAME}) gid=8004(${SC_USER_NAME}) groups=8004(${SC_USER_NAME})
+ENV SC_USER_ID 8004
+ENV SC_USER_NAME scu
+ENV SC_GROUP_ID ${SC_USER_ID}
+ENV SC_GROUP_NAME ${SC_USER_NAME}
+
+
+RUN groupadd --gid ${SC_GROUP_ID} ${SC_GROUP_NAME}
+RUN useradd -d /home/${SC_USER_NAME} --uid ${SC_USER_ID} --gid ${SC_GROUP_ID} --shell /bin/bash ${SC_USER_NAME}

```

Comment:
sur to have /bin/bash as default shell since you use /bin/sh as entrypoint?

Reply:
You are right. Fixed.